### ISSUE TITLE
I want to install this system on my local

### ISSUE THREAD

**Issue by supertfit** [mrpsion/burst-mining-system#8]
Hello, How are you?
I believe that this is silly question.
I am newbie on here.
I am gonna install this system on my local, so i followed the Readme.md
When i am following them, i had some questions.
This is what i did.
1. I install JDK 1.7 and Maven too.
2. and Run mvn package command
3. so i get mining-system-1.21.jar in target folder
Next task was 
java -jar mining-system-1.21.jar --pool.url=http://{pool.hostname}:{pool.port}

on here, i have some question.

{pool.hostname} = ? (if i use this, it can be localhost or 127.0.0.1, or my public IP )
{pool.port} = ?

That's all my question. Hope to understand my issue and help me.
Kind regard. From Jeni

**Comment by supertfit** [comment]
When i check the running on my local, i can not see the some info correctly.
Can you check what the problem is?
I attach the screenshot URL.
http://screencast.com/t/LmeSW2dcybU
Thanks.

**Comment by mrpsion** [comment]
The pool.hostname needs to be the hostname of a mining pool. I mine on http://178.62.39.204:8121
It works for both dev v2 and uray v2 pools, but not the new uray pool, he changed the API's a bit, so i havent had a chance to update it.  Currently the miners doesnt support solo mining.

**Comment by supertfit** [comment]
I bought one server and i want to make it mining the coin. In that case, i heard that i have to register that server to somewhere. Do you know how to do it?

**Comment by mrpsion** [comment]
Yeah, make sure your wallet is running and the blockchain is synced and you have created an address.  You will then need to actually have some coins in the wallet, you can send me your address and ill give you a couple, as there is a fee to register for a pool.  Once you have done all that. The go to http://localhost:8125/rewardassignment.html  In the section titles Set reward recipient:, which is at the bottom you need to put your private passphrase and the address of the pool you want to register with, if you want to use the one i mentioned previously use BURST-8NZ9-X6AX-72BK-2KFM2 as the recipient address. You should then see the transaction in your wallet. Once this has 4 confirmations you can go ahead an mine at the pool.

**Comment by supertfit** [comment]
Thanks for your replying.
My IP is 100.42.60.228
And can i ask how much i have to pay for that?

**Comment by mrpsion** [comment]
1 burst... its just the transaction fee.  I would need your burst address to send you something.

**Comment by supertfit** [comment]
my burst address is BURST-64HP-XSZ4-RZY4-D268Y

**Comment by supertfit** [comment]
Hello, mrpsion, Any news?

**Comment by mrpsion** [comment]
I sent you 100 burst 5 days ago... but from looking at your address you have generated a large amount of burst in the last month.  http://burst.cryptoport.io/acc/BURST-64HP-XSZ4-RZY4-D268Y

**Comment by mrpsion** [comment]
So upon further inspection it look like that address is actually a pool address... so unless you are running a pool... i dont think that is your address
